modules/httpd_catalog_module/unit-tests/HttpdDirScraperTest.cc

```diff
@@ -142,6 +142,7 @@ class HttpdDirScraperTest: public CppUnit::TestFixture {
     // Called before each test
     void setUp()
     {
+        if(debug) cerr << endl;
```

Comment:
You can also use DBG(...) in place of `if (debug) ...` for any unit tests that use the header in bes/modules